content/blog/2021/announcing-pycon-india.md

```diff
@@ -0,0 +1,43 @@
+Title: PyCon India is back online for 2021
+Date: 2021-02-25 17:00
+Author: Nikhil Maan (@Sc0rpi0n101)
+Category: 2021
+Slug: announce-pycon-india-2021
+Summary: Announcing the return of PyCon India as an online conference for 2021.
+
+We hope you've had a safe and healthy year. 
+
+We are very excited to announce that [**PyCon India**](https://inpycon21.netlify.app/) is back online for 2021.
```

Comment:
Don't think we should redirect to this link. 
This is temporary. Blog will break later on.

Reply:
I was thinking we would update this when we get the final website and shift to it. 

But I can remove the link if you want.

Reply:
Then how about we just point it to https://in.pycon.org/

Once it's deployed today, we can publish this. Otherwise it's more work for you to send another patch later. Upto you.

Reply:
yeah, sure. Sounds like a good idea.